chart/values.yaml

```diff
@@ -2146,7 +2146,7 @@ config:
   elasticsearch_configs:
     max_retries: 3
     timeout: 30
-    retry_timeout: 'True'
+    retry_on_timeout: 'True'
```

Comment:
Should we change this? as it will fail for elasticsearch<=7 right?

Reply:
Yes, we should because in providers.yaml it points to >8 https://github.com/apache/airflow/blob/main/airflow/providers/elasticsearch/provider.yaml#L56

Reply:
Ya, but while using someone can install the old provider version

Reply:
reverted it